### PR TITLE
Add documentation for disabling your Chatterino badge

### PR DESCRIPTION
**Pull request by killertofus**

### CODE REVIEW

**Comment by Mm2PL** [comment]
Please don't PR unrelated changes like 8293c9e. Also PRing from master (or more generally the main branch) is a bad idea because of possibility residue commits, which is what I think happened in this case.

File: docs/Help.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ Chatterino only makes a connection to the necessary APIs to fetch these emotes,
 Donate €10 to fourtf [here](https://streamelements.com/fourtf/tip) (can be done in multiple donations). You should get your badge automatically after about 5 minutes. Make sure to restart Chatterino if you can't see the badge.  
 If you still can't see it, ask fourtf on [Chatterino Discord](https://discord.gg/qq7DDxjste) for assistance.
 
+### How do I disable the supporter badge?
+login to the [manage page] (https://manage.chatterino.com/) and click on disable your Chatterino badge
```
Comment (Mm2PL) [inline docs/Help.md:72]: - New line at the end was missing.
 - Fix up grammar.
 - Stray space in markdown link syntax.

```suggestion
### How do I disable the supporter badge?
Log in to the [badge manager](https://manage.chatterino.com/) and click on *Disable My Chatterino Badge*.

```

**Review by Mm2PL — CHANGES_REQUESTED** [review]
Here is my take on this :)

**Comment by zneix** [comment]
I actually talked to fourtf weeks ago and he has specifically asked me to not officially announce this yet - he said he's not certain about whether the service can stay as it is currently, it's his microservice and until he decides, this is not officially "out" just yet.

**Comment by fourtf** [comment]
Seems fine with me

**Comment by zneix** [comment]
if you say so

**Review by pajlada — APPROVED** [review]

**Pull request merged**